wandb/sdk/launch/runner/local_container.py

```diff
@@ -29,40 +30,57 @@
 class LocalSubmittedRun(AbstractRun):
     """Instance of ``AbstractRun`` corresponding to a subprocess launched to run an entry point command locally."""
 
-    def __init__(self, command_proc: "subprocess.Popen[bytes]") -> None:
+    def __init__(self) -> None:
         super().__init__()
+        self.command_proc: Optional[subprocess.Popen] = None
```

Comment:
why not `_command_proc`?

Reply:
especially because we have a setter method in `set_command_proc` this should be `_command_proc`, and if you want a public atter make a getter method like:
```python
def command_proc(self) -> subprocess.Popen:
    # optional error handling 
    return self._command_proc
```

Reply:
Yep, meant to do that